### PR TITLE
removing jdbc-ra jars

### PR DESCRIPTION
**Pull request by ankathur**

### CODE REVIEW

**Comment by ankathur** [comment]
@glassfishrobot run ci tests please

**Comment by ankathur** [comment]
@glassfishrobot run ci tests please

File: appserver/jdbc/jdbc-ra/jdbc-ra-distribution/pom.xml [hunk 1]
```diff
@@ -64,14 +64,12 @@
             <phase>generate-sources</phase>
             <configuration>
 	      <tasks>
-                <!-- Resolving jdbc-core, jdbc30 and jdbc40 artifacts -->			
+                <!-- Resolving jdbc-core and jdbc40 artifacts -->
                 <resolveArtifact artifactId="jdbc-core" groupId="org.glassfish.main.jdbc.jdbc-ra.jdbc-core" property="core.jar" />
-                <resolveArtifact artifactId="jdbc30" groupId="org.glassfish.main.jdbc.jdbc-ra.jdbc30" property="30.jar" />
                 <resolveArtifact artifactId="jdbc40" groupId="org.glassfish.main.jdbc.jdbc-ra.jdbc40" property="40.jar" />
```
Comment (vinayvishal) [inline appserver/jdbc/jdbc-ra/jdbc-ra-distribution/pom.xml]: Is this still needed even after removal of jdbc40.jar?

File: appserver/jdbc/jdbc-ra/jdbc-ra-distribution/pom.xml [hunk 2]
```diff
@@ -64,14 +64,12 @@
...
 		<resolveArtifact artifactId="templates" groupId="org.glassfish.main.jdbc" property="templates.jar" />
 
 	        <unjar src="${core.jar}" dest="target/classes" />
-	        <unjar src="${30.jar}" dest="target/classes" />
 	        <unjar src="${40.jar}" dest="target/classes" />
```
Comment (vinayvishal) [inline appserver/jdbc/jdbc-ra/jdbc-ra-distribution/pom.xml]: Is this still needed even after removal of jdbc40.jar?

File: appserver/jdbc/jdbc-ra/jdbc40/pom.xml [hunk 3]
```diff
@@ -73,9 +73,9 @@
                   Use JDBC 4.0 API regardless of which JDK we are building with.
                 -->
                 <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
+              <!--  <configuration>
```
Comment (vinayvishal) [inline appserver/jdbc/jdbc-ra/jdbc40/pom.xml]: Instead of commenting it out, it can be removed.

File: appserver/jdbc/jdbc-ra/pom.xml [hunk 4]
```diff
@@ -89,15 +89,15 @@
...
+        <role>lead</role>
 	<role>developer</role>
       </roles>
     </developer>
   </developers>
 
   <modules>
     <module>jdbc-core</module>
-    <module>jdbc30</module>
+    <!--<module>jdbc30</module>-->
```
Comment (vinayvishal) [inline appserver/jdbc/jdbc-ra/pom.xml]: we can remove it instead of commenting it out.

File: appserver/jdbc/jdbc-ra/pom.xml [hunk 5]
```diff
@@ -89,15 +89,15 @@
...
     <module>jdbc40</module>
```
Comment (vinayvishal) [inline appserver/jdbc/jdbc-ra/pom.xml]: shouldn't this too be removed since we are removing jdbc40.jar as well.

**Review by vinayvishal — CHANGES_REQUESTED** [review]

File: appserver/jdbc/jdbc-ra/jdbc40/pom.xml (hunk 3 above)
Comment (ankathur) [reply appserver/jdbc/jdbc-ra/jdbc40/pom.xml]: Done.

File: appserver/jdbc/jdbc-ra/pom.xml (hunk 4 above)
Comment (ankathur) [reply appserver/jdbc/jdbc-ra/pom.xml]: Done.

**Review by vinayvishal — APPROVED** [review]
Looks good.

**Pull request merged**